src/prefect/orion/schemas/core.py

```diff
@@ -467,6 +468,17 @@ class TaskRun(ORMBaseModel):
     def set_name(cls, name):
         return name or generate_slug(2)
 
+    @validator("cache_key")
+    def validate_cache_key_length(cls, cache_key):
+        if (
+            cache_key
+            and len(cache_key) > PREFECT_ORION_TASK_CACHE_KEY_MAX_LENGTH.value()
+        ):
+            raise ValueError(
+                "Cache key exceeded maximum allowed length of {PREFECT_ORION_TASK_CACHE_KEY_MAX_LENGTH} characters."
```

Comment:
Probably going to exceed line length limits

```suggestion
                f"Cache key exceeded maximum allowed length of {PREFECT_ORION_TASK_CACHE_KEY_MAX_LENGTH.value()} characters."
```